frontend/ui/src/pages/Channels/Providers/Twilio/TwilioConnect.tsx

```diff
@@ -17,6 +17,7 @@ type TwilioConnectProps = {
   source: string;
   pageTitle: string;
   buttonText: string;
+  infoLink: string;
 } & ConnectedProps<typeof connector> &
```

Comment:
for Twilio, added the infoLink as a props so that each source can use a different link  while using the same component